.travis.yml

```diff
@@ -81,6 +84,8 @@ after_script:
     - echo $TRAVIS_COMMIT_LOG
 
 jobs:
+  exclude:
+    - os: osx
```

Comment:
this would not be required either if not mentioning it in the `os` section globally